src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/search-button-field.html

```diff
@@ -1,7 +1,5 @@
-{# Behaves the same as `search-button.html` but looks more like a search field.
-  #
-  # As this function will only work when JavaScript is enabled, we add it through JavaScript.
-  #}
+{# Behaves the same as `search-button.html` but looks more like a search field. #}
+{# As this function will only work when JavaScript is enabled, we add it through JavaScript. #}
```

Comment:
You could keep the multiline comments if you want by cutting the extracted comment at the first new line? (Similar to a docstring where the first line is the summary)

Reply:
I though about it but then I also thought that the list would become overwelming for users. 
The second comment is not a "docstrng" it's the justification for the `script` wrapper. I don't think user should be expose to this level of information.